packages/paste-core/components/textarea/stories/textarea.stories.tsx

```diff
@@ -25,12 +25,13 @@ export const Textarea = (): React.ReactNode => {
       <Label htmlFor={uid}>Label</Label>
       <TextArea
         id={uid}
+        aria-describedby={`help-text-${uid}`}
```

Comment:
`useUID()` is the generator, `uid` is the output of the generation, so I think this is showing best practices.

Reply:
Its not the generator, its a variable - `const uid = useUID();` is on line 23.

The ids match correctly in storybook.

Reply:
Clarification: I am pointing out that the id for the text area and the label are the same for all of the stories. If there are multiple in the same page, this would be a problem.

It looked to me like there are multiple per page, if not disregard.

Reply:
The way `useUID` works as a hook in the component, is that if you do have multiple instances of the component on the same page it will generate a different ID per instance.